lib/components/artist/shows/small_image_show.js

```diff
@@ -16,10 +16,12 @@ class SmallImageShow extends React.Component {
 
   render() {
     const show = this.props.show;
+    const image = show.meta_image;
+    const imageURL = image && image.cropped.url;
     return (
       <TouchableWithoutFeedback onPress={this.handleTap.bind(this)} >
         <View style={styles.container}>
-          <OpaqueImageView imageURL={show.meta_image.cropped.url} style={styles.image} />
+          <OpaqueImageView imageURL={imageURL} style={styles.image} />
           <ShowMetadata show={show} />
         </View>
       </TouchableWithoutFeedback>
```

Comment:
@sarahscott These three components are all the same in terms of the components they use and their JSX code, the only difference is styling. I think that can be refactored so that only the styling needs to be specified for each individual variant, but the JSX can stay the same and in 1 source file.